src/foam/u2/svg/graph/ArrowDisplaceCellsPlacementPlan.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
```

Comment:
Is this a new file or just renamed?

Reply:
yes this is new - updated the copyright header